db/migrate/20150714151223_rename_chef_proxy.rb

```diff
@@ -0,0 +1,9 @@
+class RenameChefProxy < ActiveRecord::Migration
+  def up
+    chef_feature = Feature.find_by_name("Chef Proxy")
+    chef_feature.update_attribute(:name, "Chef") if chef_feature
```

Comment:
If foreman_chef's loaded, this might lead to a duplicate - should we delete Chef Proxy if it's there, leaving the plugin to define Chef?